### ISSUE TITLE
Trying to open empty pages throws an error

### ISSUE THREAD

**Issue by Mackiovello** [StarcounterApps/Content#36]
### Steps to reproduce

1. Go to `/content/cms`
2. Click on the `CREATE` button
3. Enter an URL for the new item, `/something` for example
4. Click the `SAVE` button
5. Go to `/content/dynamic/something`

### Expected result

An empty page should show or the user is stopped with a notice

### Actual result

The following error is shown:

```
System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. Parameter name: Can not find referenced Html file: "/content/dynamic/SomethingPage.html" at Starcounter.HtmlFromJsonProvider.ProvideFromFilePath[T](String filePath) in C:\TeamCity\BuildAgent\work\sc-4704\Level1\src\Starcounter\Middleware\HtmlFromJsonProvider.cs:line 58 at Starcounter.HtmlFromJsonProvider.Invoke(MimeProviderContext context, Action next) in C:\TeamCity\BuildAgent\work\sc-4704\Level1\src\Starcounter\Middleware\HtmlFromJsonProvider.cs:line 53 at Starcounter.Internal.JsonMimeConverter.Convert(Request request, IResource before, MimeType mimeType, MimeType& resultingMimeType) in C:\TeamCity\BuildAgent\work\sc-4704\Level1\src\Starcounter.Apps.JsonPatch\BuiltInRestHandlers\JsonMimeConverter.cs:line 18 at Starcounter.Json.AsMimeType(MimeType mimeType, MimeType& resultingMimeType, Request request) in C:\TeamCity\BuildAgent\work\sc-4704\Level1\src\Starcounter.XSON\Class Json\Json.IResource.cs:line 45 at Starcounter.Response.ConstructFromFields(Request req, Byte[] givenBuffer) in C:\TeamCity\BuildAgent\work\sc-4704\Level1\src\Starcounter.Internal\Http\Response.cs:line 809 at Starcounter.Request.SendResponse(Response resp, Byte[] serializationBuf) in C:\TeamCity\BuildAgent\work\sc-4704\Level1\src\Starcounter.Internal\Http\Request.cs:line 1241 at Starcounter.Internal.AppsBootstrapper.ProcessExternalRequest(Request req) in C:\TeamCity\BuildAgent\work\sc-4704\Level1\src\Starcounter.Apps.JsonPatch\AppsBootstrapper.cs:line 811 ParamName=Can not find referenced Html file: "/content/dynamic/SomethingPage.html" HResult=-2146233086 More information about this error may be available in the server error log.
```

##### Starcounter version 

Version=2.3.0.6278
@a2106f61c3d7d9f4cd98fa5b8e1ad11f02eb09e5
 
This issue has existed, at least, from @66f1cc3acb0c3b29d51fd66ff2047b24ada16fec

**Comment by Mackiovello** [comment]
link #35

**Comment by Mackiovello** [comment]
Fixed with 		78727d7. Closing
